cdap-client/src/main/java/io/cdap/cdap/client/ScheduleClient.java

```diff
@@ -263,4 +266,7 @@ private List<ScheduleDetail> doList(WorkflowId workflow)
     return objectResponse.getResponseObject();
   }
 
+  String getEncodedScheduleName(String scheduleName) throws UnsupportedEncodingException {
+    return URLEncoder.encode(scheduleName, StandardCharsets.UTF_8.toString()).replace("+", "%20");
```

Comment:
URL encoder encodes `' '` character to `'+'` instead of `%20`.

ref: https://stackoverflow.com/questions/4737841/urlencoder-not-able-to-translate-space-character

Reply:
Why is it a problem? This is WAI.

Reply:
Let's check why. `+` is a valid encoding for space in URI, so decoder should handle it